docs/dev/table/sql.md

```diff
@@ -1842,6 +1842,16 @@ RPAD(text string, len integer, pad string)
     <tr>
       <td>
         {% highlight text %}
+REGEX_EXTRACT(str string, regex string, extractIndex integer)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Returns the string str extracted using specified regex pattern and index. If str or regex is null, returns null. E.g. <code>REGEX_EXTRACT('foothebar', 'foo(.*?)(bar)', 2)</code> returns <code>bar</code>.</p>
```

Comment:
Also similar to SQL array, might be good to point out that the index starts with 1.

Reply:
for`REGEX_EXTRACT `,  it can pass 0, that means extract all.

Reply:
it just returns the extractIndex-th value of match group array, just one element not an array, refers to : https://cwiki.apache.org/confluence/display/Hive/LanguageManual+UDF (please search 'regex_extract')

Reply:
Thanks for the explanation. Sorry I was a bit confused on the wording in the doc. Maybe similar to Hive, we can add this line to the doc: 
```
See docs/api/java/util/regex/Matcher.html for more information
```
This way it is immediately clear, what do you think?

Reply:
I agree this idea, but the path `docs/api/java/util/regex/Matcher.html`, how to access? It's not a complete URL.